src/github.com/getlantern/http-proxy-extensions/mimic_test.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const target = "test-data/apache-2.4.7-ubuntu14.04.raw"
 const template = "test-data/apache-2.4.7-ubuntu14.04.tpl"
-const current = "test-data/chained-server.raw"
+const current = "test-data/http-proxy.raw"
```

Comment:
Sorry, I changed that in the .gitignore file. I thought it was automatically generated in the build process, derived from the binary name!